gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -328,6 +329,35 @@ export default function NxDrawerPage() {
         </NxTile.Content>
       </NxTile>
 
+      <NxTile>
+        <NxTile.Header>
+          <NxTile.HeaderTitle>
+            <NxH2>NxDrawer With NxToast Example</NxH2>
+          </NxTile.HeaderTitle>
+        </NxTile.Header>
+        <NxTile.Content>
+          <NxP>
+            An example of <NxCode>NxDrawer</NxCode> with <NxCode>NxToast</NxCode>. If a toast is already open and
+            then a drawer is opened the toast appears on top of the drawer. This is the same if the toast is opened
+            from a button inside the drawer. Both the drawer and toast will close if a toast is closed.
```

Comment:
Since you mention both scenarios, maybe include a toast button outside of the drawer inside NxPageMain to see both behaviours

Reply:
> Since you mention both scenarios, maybe include a toast button outside of the drawer inside NxPageMain to see both behaviours

Added it here: abf354e3be917cf7923530d67ed955b645ea326a